CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Custom Headers to be added in the UI
+
+### Added
+- Removed search term path from the UI
```

Comment:
```suggestion
### Added
- Custom Headers to be added in the UI
- Removed search term path from the UI
```

Reply:
não seria melhor usar o `### Removed`?

Reply:
I keep @vmourac-vtex recommendation, that's what I was trying to do, probably misspelled the changelog, thanks folks